frontend/Dockerfile.prod

```diff
@@ -0,0 +1,56 @@
+# Install dependencies only when needed
+FROM node:18.15-bullseye AS deps
+
+WORKDIR /app
```

Comment:
I'm not very familiar with Docker. I just want to double check with you that this folder is in the virtual machine and not the repository because we have no such folder there inside `frontend`.

Reply:
that's fine, yes it's not expected to be in the repository. It is used to set the context for building and will be created if not already there.